run_tests.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+case "$@" in
+	*-k*) partial_test=1 ;;
```

Comment:
I couldn't get this flag to work, but maybe just because I don't understand what it is supposed to do. Could you include an example usage with this flag?

Reply:
`-k` is passed through to pytest. Pytest will run only tests that match the specified string. https://docs.pytest.org/en/latest/usage.html#specifying-tests-selecting-tests

When using this argument, the coverage output is not useful—Since most tests are skipped, the coverage is incorrect and the output is just noise.

I like to omit `--cov` when running a subset of tests to keep the output easy to read. It fits my test-driven development workflow, but perhaps it's just a personal quirk. I have no objection to moving `--cov` to `pytest.python3.ini` if that is the team's preference, but just want to point out that it makes TDD less convenient.